src/lib/process/api.ts

```diff
@@ -12,6 +12,11 @@ const isProgramValid = (program: Process.Program, out = { ops: 0, restrictions:
   return out.ops === out.restrictions
 }
 
+let lastSubmittedNonce: number = -1
+export function resetLastSubmittedNonce() {
```

Comment:
is this only for the tests?

Reply:
for now yes. I can forsee conditions where we'd need to do this elsewehere